axis/main.f95

```diff
@@ -0,0 +1,43 @@
+program resonant_axis_calculator
```

Comment:
Just FYI: if I run the command without the arguments, it shows the error. But how can I find the arguments I should put?